.changeset/fluffy-dragons-melt.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cmpsr/ui-storybook": patch
+---
+
+COM-1167 Fixes delete-storybook github action
```

Comment:
we should remove this file, this change doesn't need a release and this will add noise to the changeset